### ISSUE TITLE
Plus button missing accessibility label

### ISSUE THREAD

**Issue by jleach** [bcgov/bc-wallet-mobile#2016]
The "+" button on the messaging screen is missing an accessibility label and shows as a question mark on iOS. Found in 1.0.17 1744

![Image](https://github.com/bcgov/bc-wallet-mobile/assets/390891/24bc6e71-f1c3-4b52-8059-841a4e7ba4f4)

**Comment by jeznorth** [comment]
Fixed as part of #1867.
